Sources/MapboxCoreNavigation/RouterDelegate.swift

```diff
@@ -129,6 +129,16 @@ public protocol RouterDelegate: AnyObject, UnimplementedLogging {
      */
     func router(_ router: Router, didFailToTakeAlternativeRouteAt location: CLLocation?)
     
+    /**
+     Called when router has automatically switched to the coincide online route.
+     
+     - note: `LegacyRouteController` will never do that.
```

Comment:
The `note` command is intended for things that require a bit more attention. This is more for an FYI.

```suggestion
     `LegacyRouteController` never calls this method, because it cannot generate routes on the device.
```